src/js/reducers/sessionReducer.jsx

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+
+import { FETCH_SESSION_INFO, CHANGE_CURRENT_LOCATION } from '../actions/types';
+
+const initialState = {
+  currentLocation: {
+    id: '',
+    name: '',
+    hasBinLocationSupport: true,
+    locationType: { description: '', locationTypeCode: '' },
+    isSuperuser: false,
```

Comment:
The isSuperuser property should not be in the currentLocation, add it directly to the initial state, because it accessed like that below